datafusion/functions/src/core/struct.rs

```diff
@@ -73,12 +73,6 @@ impl StructFunc {
     }
 }
 
-impl Default for StructFunc {
-    fn default() -> Self {
-        Self::new()
-    }
-}
-
 impl ScalarUDFImpl for StructFunc {
```

Comment:
Last PR I make StructFunc pub so I add the Default trait, but since https://github.com/apache/arrow-datafusion/pull/9546#discussion_r1519548725 it will not be needed.